engine/src/main/java/org/destinationsol/game/projectile/Projectile.java

```diff
@@ -263,9 +267,7 @@ public boolean shouldCollide(SolObject object, Fixture fixture, FactionManager f
             }
             return true;
         }
-        if (object instanceof Projectile) {
```

Comment:
We don't need this anymore?

Reply:
That was for checking projectiles from the same faction don't hit each other. So no, we probably don't need that.